versioneer.py

```diff
@@ -1786,8 +1786,6 @@ def run(self):
         _build_py = cmds["build_py"]
     elif "setuptools" in sys.modules:
         from setuptools.command.build_py import build_py as _build_py
-    else:
```

Comment:
I'm just curious about what changed to make this line safe to delete, was this a change from the upstream versioneer.py? (Same question for a few of the `else:` removals further down too.

Reply:
Yeah, recent versions of versioneer don't have those lines. 
We install setuptools in `requirements-dev.txt` and with that package we don't need to worry about the fall back to distutils which no longer ships with Python as of 3.12.